sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -95,6 +96,8 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     );
     programOffering.offeringStatus = offeringValidation.offeringStatus;
     programOffering.creator = { id: userId } as User;
+    // When creating a new offering, parent id and the primary id are the same.
+    programOffering.parentOffering = programOffering;
```

Comment:
Is this still effective with using the insert and update late? I believe we can remove it.